vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/payment/payment_server.h

```diff
@@ -34,6 +35,8 @@ class PaymentServer {
 
   payment::PostVotes* post_votes() const;
 
+  payment::PostTransactionGemini* post_transaction_gemini() const;
```

Comment:
this really needs to be refactored into common base classes (same with `uphold()`, `gemini()`, etc... adding separate methods for each wallet type all over the place adds to fragile code like https://github.com/brave/brave-core/pull/8566/files#r664763451

Reply:
Follow-up issue here: https://github.com/brave/brave-browser/issues/16792